nav2_system_tests/src/system/test_system_launch.py

```diff
@@ -76,8 +76,8 @@ def generate_launch_description():
             parameters=[params_file]),
 
         launch_ros.actions.Node(
-            package='dwb_controller',
-            node_executable='dwb_controller',
+            package='nav2_controller',
+            node_executable='controller_server',
```

Comment:
I think you missed to change this line in `nav2_simulation_launch.py` unless its just not showing up on the github view

Reply:
@SteveMacenski I just checked now in ```nav2_bringup/launch/nav2_simulation_launch.py```
The ```dwb_controller``` had already been renamed. 
Or are you referring to somewhere else?

Reply:
Ah ok, github then